js/ascendex.js

```diff
@@ -1263,7 +1263,7 @@ module.exports = class ascendex extends Exchange {
     async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         await this.loadMarkets ();
         await this.loadAccounts ();
-        const defaultAccountCategory = this.safeString (this.options, 'account-category', this.defaultType);
+        const defaultAccountCategory = this.safeString (this.options, 'account-category');
```

Comment:
Technically, the defaultAccountCategory must be reworked to defaultType, we should only use handleMarketTypeAndParams here, replacing the old account-category-code with it.